content/docs/pipelines/pipelines-deploy-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Deploy Alternatives"
+description = "Alternative approaches to deploy lightweight Kubeflow Pipelines to a cluster"
+weight = 15
++++
+
+Despite [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+Kubeflow Pipelines' lite deployment. Pipelines has a kustomize manifests for
```

Comment:
Suggestion for improved clarity: Change this:
"Pipelines has a kustomize manifests for this. You can follow the instructions below to deploy it in an existing Kubernetes cluster."

To this:
"Follow the instructions below to deploy Kubeflow Pipelines in an existing cluster, using the supplied kustomize manifest."